hack/terraform/aws/delete_dependency.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/bash
+# Copyright 2022 Antrea Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+cat $HOME/tmp/terraform-aws/terraform.tfstate | jq > tmp.json
+vpc_id="$( jq '.outputs.vpc_id.value' tmp.json | sed 's/\"//g' )"
+region="$( jq '.outputs.region.value' tmp.json | sed 's/\"//g' )"
+sgs="$( aws ec2 describe-security-groups --region $region --filters Name=vpc-id,Values=$vpc_id | jq '.SecurityGroups[].GroupId' )"
+tf_created="$( jq ".resources[].instances[].attributes.id" tmp.json )"
```

Comment:
Same thing here. Get these info in terraform and pass them in as param. Parsing state file is hacky and lets try avoid it.

Reply:
@shenmo3 I think vpc_id and region are easy to get, but because of the tf_created are all the IDs of all the instances(not only SG),  here is the only way that we can get the SGs on the VMs, how can we get that ID in .tf file?